doc/en/docguide/source/install.rst

```diff
@@ -60,8 +56,7 @@ Ubuntu
 Mac OS X
 --------
 
-Installing Sphinx on Mac OS X is nearly identical to installing Sphinx on a 
-Linux system. 
+Installing Sphinx on Mac OS X is nearly identical to installing Sphinx on a Linux system. 
```

Comment:
I don't think this line is correct anymore, or at least it won't be if you are just using apt to install sphinx on ubuntu